src/main.ts

```diff
@@ -1,8 +1,14 @@
 import {NestFactory} from '@nestjs/core';
 import {AppModule} from './app.module';
+import {SwaggerModule, DocumentBuilder} from '@nestjs/swagger';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
+  const config = new DocumentBuilder().setTitle('Shop').setDescription('The Shop API description').setVersion('1.0').addTag('todo').build();
```

Comment:
Why `todo` tag?

Reply:
Copied from previous file and didn`t see it. I have already removed it, thanks.